src/platform/Beken/Logging.cpp

```diff
@@ -0,0 +1,69 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
```

Comment:
Please fix copyright headers (2020 -> 2022) in all files.

Reply:
Ok, Thanks.